patches/0004-Use-crypto-backends.patch

```diff
@@ -198,6 +199,18 @@ index ed0fbf3d53d75b..8111b143a1295b 100644
  	tg := testgo(t)
  	defer tg.cleanup()
  	tg.parallel()
+diff --git a/src/cmd/go/script_test.go b/src/cmd/go/script_test.go
+index 390a36723787f4..105bac4e1c627b 100644
+--- a/src/cmd/go/script_test.go
++++ b/src/cmd/go/script_test.go
+@@ -254,6 +254,7 @@ func scriptEnv(srv *vcstest.Server, srvCertFile string) ([]string, error) {
+ 		"HGRCPATH=",
+ 		"GOTOOLCHAIN=auto",
+ 		"newline=\n",
++		"OPENSSL_FORCE_FIPS_MODE=" + os.Getenv("OPENSSL_FORCE_FIPS_MODE"), // useful for testing on Mariner 2.
```

Comment:
I feel like this would fit better in `extraEnvKeys`. (Unless there's a reason that it must be set to empty string even if the env var is unassigned?)

Reply:
Good catch, will do!